SLObjectiveCRuntimeAdditions/SLObjectiveCRuntimeAdditions.m

```diff
@@ -48,8 +48,8 @@ - (BOOL)implementInstanceMethodNamed:(SEL)selector types:(const char *)types imp
     NSParameterAssert(blockImplementation);
     NSParameterAssert(types);
 
-    __assert_unused SLBlockDescription *blockDescription = [[SLBlockDescription alloc] initWithBlock:blockImplementation];
-    __assert_unused NSMethodSignature *methodSignature = [NSMethodSignature signatureWithObjCTypes:types];
+    __unused SLBlockDescription *blockDescription = [[SLBlockDescription alloc] initWithBlock:blockImplementation];
+    __unused NSMethodSignature *methodSignature = [NSMethodSignature signatureWithObjCTypes:types];
```

Comment:
It throws me an ‘Use of undefined identifier __assert_unused'

On Sat, Oct 24, 2015 at 4:24 PM, Oliver Letterer <notifications@github.com>
wrote:

>> @@ -48,8 +48,8 @@ - (BOOL)implementInstanceMethodNamed:(SEL)selector types:(const char *)types imp
>>      NSParameterAssert(blockImplementation);
>>      NSParameterAssert(types);
>>  
>> -    __assert_unused SLBlockDescription *blockDescription = [[SLBlockDescription alloc] initWithBlock:blockImplementation];
>> -    __assert_unused NSMethodSignature *methodSignature = [NSMethodSignature signatureWithObjCTypes:types];
>> +    __unused SLBlockDescription *blockDescription = [[SLBlockDescription alloc] initWithBlock:blockImplementation];
>> +    __unused NSMethodSignature *methodSignature = [NSMethodSignature signatureWithObjCTypes:types];
> This should still be __assert_unused to avoid warnings with NS_BLOCK_ASSERTIONS and CocoaPods.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/OliverLetterer/SLObjectiveCRuntimeAdditions/pull/4/files#r42934428